### PR TITLE
Card and tagline changes to accommodate new homepage

### PR DESCRIPTION
**Pull request by contolini**
## Changes

- Increase font-size of breakout card links
- Increase the spacing under card group headings
- Decrease the font size of XL taglines

## Testing

1. Compare the production [cards page](https://cfpb.github.io/design-system/patterns/cards#topic-card-group) to the PR preview link below.
2. The breakout card links should be larger.
3. Card group headings should have more space underneath
4. The XL tagline text should be slightly smaller.

## Screenshots

| before | after |
|--------|-------|
| before  | after |
| before | after |
| before  | after |

## Checklist

- [ ] PR has an informative and human-readable title
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code can be automatically merged (no conflicts)
- [ ] Code follows the standards laid out in the [CFPB development guidelines](https://github.com/cfpb/development)
- [ ] Passes all existing automated tests
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future todos are captured in comments
- [ ] Visually tested in supported browsers and devices (see checklist below :point_down:)
- [ ] Project documentation has been updated
- [ ] Reviewers requested with the [Reviewers tool](https://help.github.com/articles/requesting-a-pull-request-review/) :arrow_right:

### Browsers

- [ ] Chrome on desktop
- [ ] Firefox
- [ ] Safari on macOS
- [ ] Edge
- [ ] Internet Explorer 9, 10, and 11
- [ ] Safari on iOS
- [ ] Chrome on Android

### Accessibility

- [ ] Keyboard friendly
- [ ] Screen reader friendly

### Other

- [ ] Is useable without CSS
- [ ] Is useable without JS
- [ ] Flexible from small to large screens
- [ ] No linting errors or warnings
- [ ] JavaScript tests are passing

### CODE REVIEW

File: packages/cfpb-typography/src/atoms/tagline.less [hunk 1]
```diff
@@ -36,7 +36,7 @@
             background-image: url( 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFAAAAAqCAMAAAATdiw4AAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAADlQTFRF////sxlC2YyhCjFhKUt1R2WJGT5rhZiwOFh/Zn6co7LEdYum0djhV3GS8PL1wszYsr/O4OXrlKW6gKQKnwAAAUhJREFUeNrslM2OwzAIhN2OHf/HSd//YXfYqGroZWWUS9XlgPgOGRmY4IAFjOXIr/Qsb5PhkEbglzHy4zAS3tkg6CIFWqNAdCKoeV7Qp4SIEJhS8iFohpsNGVVxGciuAH7f/ZlhEUzo0TNiZ1mrZoNgXuVlrcnLVnmZ5vtkcIYj0x6lMOXhl0WzYSmM2DrQG7frS/Fnhk2wrlTxa2U5hmZDy6kE9lkruw0lea/ZsJTkHjTyvtPIj8PYiucF6WVk9M7062rNNmNvjuOqbpPByfhebFpKQj6MnFnGqNkgWNftaeRNFvvGhuMwKj28bUz1MPaZbTOMTX65s7GfDKOgEyPLHURrmi2CD7DR3tmslF6z4U8JcvcOIxfH6//GhqV0qiBzaghdWtU8LRj+iOmWvzBuF8cHCH5h3C+OD9jy5S3/H4cvOA4/AgwAabgYexE/bU4AAAAASUVORK5CYII=' );
         }
 
-        font-size: unit( 26px / @base-font-size-px, rem );
+        font-size: unit( 22px / @base-font-size-px, rem );
```
Comment (anselmbradford) [inline packages/cfpb-typography/src/atoms/tagline.less:39]: I think this was only to decrease at mobile size?

File: packages/cfpb-layout/src/molecules/card.less [hunk 2]
```diff
@@ -269,6 +269,11 @@
 
         // Arguments: default, `:visited`, `:hover`, `:focus`, and `:active` states.
         .u-link-card__colors( @pacific, @teal, @pacific-dark, @pacific, @navy );
+
+        // Breakout cards have larger links.
+        .m-card_footer > span {
+            .heading-3();
```
Comment (anselmbradford) [inline packages/cfpb-layout/src/molecules/card.less:275]: We need to add margin or padding under this to make it equal-ish on the top and bottom of the text, ye?

File: packages/cfpb-typography/src/atoms/tagline.less (hunk 1 above)
Comment (contolini) [reply packages/cfpb-typography/src/atoms/tagline.less:39]: Ah yesh 👍

File: packages/cfpb-layout/src/molecules/card.less (hunk 2 above)
Comment (contolini) [reply packages/cfpb-layout/src/molecules/card.less:275]: Mmmm yeah I posted a screenshot over in GHE but in the DS it definitely looks like it's running too close to the bottom border. I'll reference the comp and adjust spacing accordingly.

File: packages/cfpb-layout/src/molecules/card.less [hunk 3]
```diff
@@ -254,7 +254,7 @@
         }
 
         .m-card_footer {
-            margin-top: ( @card_img_height / 2 ) + @grid_gutter-width;
+            margin-top: ( @card_img_height / 2.2 ) + @grid_gutter-width;
```
Comment (anselmbradford) [inline packages/cfpb-layout/src/molecules/card.less:257]: I'm going to change this to subtract 6px so it ends up a whole pixel value and is not dependent on the image height.

**Comment by anselmbradford** [comment]
Mergin' so we can get this into testin'

**Pull request merged**